www/ssc/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
```

Comment:
MacPorts has utilities available for projects hosted on Github.
The main one is the `github` portgroup.  Think of a portgroup as an "importable" that provides new or shared functionality.

Once you specify the `github` portgroup, you will then be able to use `github.setup`:
```
PortSystem              1.0
PortGroup               github 1.0
PortGroup               cmake 1.1

github.setup           devongarde ssc 0.1.55
github.tarball_from    archive
```

The `github.setup` stanza will implicitly set `name`, `version`, `distname`, `distfiles`, `extract.*`, and `master_sites`, allowing you to remove all of those lines.

Reply:
You will also want to add a `revision` field.  `revision` is what you increment if you want to make updates to a port whose version number isn't changing:

```
github.setup           devongarde ssc 0.1.55
github.tarball_from    archive
revision               0
```